setup.cfg

```diff
@@ -25,6 +25,8 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
+    deephaven
+    deephaven-jpy
```

Comment:
@devinrsmith if I add this I get errors when installing the wheel:
```Processing /tmp/deephaven_plugin_matplotlib-0.0.1.dev7-py3-none-any.whl
Collecting deephaven-plugin
  Using cached deephaven_plugin-0.2.0-py3-none-any.whl (7.7 kB)
ERROR: Could not find a version that satisfies the requirement deephaven-jpy (from deephaven-plugin-matplotlib) (from versions: none)
ERROR: No matching distribution found for deephaven-jpy
```

What's the correct course of action here?

Reply:
How / where are you trying to install it? It should only be installed in a location that already has deephaven setup, yeah?